contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -85,6 +92,12 @@ contract PBHEntryPointImplV1 is IPBHEntryPoint, WorldIDImpl, ReentrancyGuardTran
     /// @param pbhGasLimit The gas limit for a PBH multicall transaction.
     event PBHGasLimitSet(uint256 indexed pbhGasLimit);
 
+    /// @notice Emitted when the nullifier hashes are spent.
+    ///
+    /// @param builder The address of the builder that spent the nullifier hashes.
+    /// @param nullifierHashes The nullifier hashes that were spent.
+    event NullifierHashesSpent(address indexed builder, uint256[] nullifierHashes);
```

Comment:
nit: Consider renaming to `PBHNullifiersSpent` or similar for readability. While minor this change can help denote what the nullifier is for.